### PR TITLE
Fixes #404 -- added checklist for open zaak post install

### LINKED ISSUE #404: As installer, I want to have a checklist on how to check if Open Zaak works as expected
... so I can leave a municipality with confidence.

**Description**
Document a post-install-and-setup checklist and/or a default simple Postman collection to retrieve a list of Zaken and create a Zaak.

A `manage.py raven test` should also be part of the checklist.

### PR DESCRIPTION
**Pull request by stevenbal**
issue #404 

I will make separate pull requests for the notification management command and view-config changes

### CODE REVIEW

File: docs/installation/index.rst [hunk 1]
```diff
@@ -44,3 +44,68 @@ Guides
...
+
+Post-install checklist
+----------------------
+
+After Open Zaak has been installed successfully, go through the following checklist
+to see if the software works as expected:
+
+**Check configuration**
+
+Check the configuration page for Open Zaak, accessible at the url ``{{base_url}}/view-config/``.
```
Comment (sergei-maertens) [inline docs/installation/index.rst]:
```suggestion
Check the configuration page for Open Zaak, accessible at the url https://open-zaak.gemeente.nl/view-config/.
```
We've been using that placeholder domain for examples most

File: docs/installation/index.rst [hunk 2]
```diff
@@ -44,3 +44,68 @@ Guides
...
+**Run check management commands**
+
+If Sentry was set up for Open Zaak, make sure to run the following command to
+ensure that logging to Sentry will work as expected:
+
+.. code-block:: shell
+
+    python src/manage.py raven test
+
+Furthermore, to verify that notifications can be received by the URL as defined
```
Comment (sergei-maertens) [inline docs/installation/index.rst]: Please add that the `test` kanaal must exist in the NRC you're using, see https://github.com/open-zaak/open-zaak/pull/442/files#r371121413

**Comment by joeribekker** [comment]
Other than Sergei's comment, looks good.

File: docs/installation/index.rst (hunk 2 above)
Comment (sergei-maertens) [reply docs/installation/index.rst]: Added a note

**Pull request merged**